xwayland.ml

```diff
@@ -951,8 +958,9 @@ let listen_x11 ~selection t =
       Log.info (fun f -> f "PropertyNotify: %a/%a %s" X11.Window.pp window (X11.Atom.pp x11) atom
                    (match state with `NewValue -> "has new value" | `Deleted -> "deleted"));
       Fiber.fork ~sw:t.sw (fun () ->
-          let wm_name = intern t "WM_NAME" in
-          if atom = wm_name then (
+          let wm_name = intern t "WM_NAME"
+          and net_wm_name = intern t "_NET_WM_NAME" in
+          if atom = wm_name || atom = net_wm_name then (
             match Hashtbl.find_opt t.paired window with
             | Some { xdg_role = `Toplevel toplevel; _ } ->
               let title = X11.Property.get_string x11 window wm_name |> Option.value ~default:"<untitled>" in
```

Comment:
This isn't quite right - it always uses `wm_name`. It should only use that if the NET property isn't being used.

Reply:
yes, that's the comment i made earlier :p this would require tracking if `_NET_WM_NAME` is set, which i haven't yet done.

Reply:
It doesn't work at all, because it uses wm_name even if the property being set is net_wm_name. I've pushed a fix for that to your branch. If you're happy with that, it's ready to merge!

Reply:
Yeah, that looks good to me! Probably better than I would've written myself :p